src/App.tsx

```diff
@@ -1,14 +1,51 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useState, useEffect } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todo, setTodo] = useState<Todo | null>(null);
```

Comment:
you've used it only once to get todos in array and [ass it to load. maybe you can use only load without extra states